catalog/core/catalog-core-commands/src/main/java/org/codice/ddf/commands/catalog/ImportCommand.java

```diff
@@ -196,6 +196,9 @@ protected Object executeWithSubject() throws Exception {
 
                 entry = zipInputStream.getNextEntry();
             }
+        } catch (Exception e) {
+            LOGGER.error("Exception while importing metacards", e);
```

Comment:
Since this is at error level, can you just provide some more info on what someone might need to do to resolve this?